plugins/restlet1x/nexus-restlet1x-plugin/src/main/java/org/sonatype/nexus/rest/repositorystatuses/RepositoryStatusesListPlexusResource.java

```diff
@@ -76,6 +84,15 @@ public PathProtectionDescriptor getResourceProtection() {
   public Object get(Context context, Request request, Response response, Variant variant)
       throws ResourceException
   {
+    if (adminOnly) {
+      Subject subject = ThreadContext.getSubject();
+      // skip building reply unless user has admin access (ie. can update status)
+      if (subject == null || !subject.isPermitted("nexus:repostatus:update")) {
```

Comment:
where is this perm added?  I was expecting subject.hasRole("nx-admin").  Perm check may be fine too, but depends on how its configured...